src/lib/converter/nodes/module.ts

```diff
@@ -24,7 +24,9 @@ export class ModuleConverter extends ConverterNodeComponent<ts.ModuleDeclaration
     convert(context: Context, node: ts.ModuleDeclaration): Reflection | undefined {
         const reflection = context.isInherit && context.inheritParent === node
             ? <DeclarationReflection> context.scope
-            : createDeclaration(context, node, ReflectionKind.Namespace);
+            : createDeclaration(context, node, node.name.kind === 10
```

Comment:
Using the kind is fine (that's all the function does) but the inline constant could break across different TS versions. We should instead use the enum.

```suggestion
            : createDeclaration(context, node, node.name.kind === ts.SyntaxKind.StringLiteral
```

Reply:
I should have time later today or tomorrow to pull this down and fix if you don't have time.